catalog/core/catalog-core-subscriptionstore/subscriptionstore-impl/pom.xml

```diff
@@ -42,6 +42,26 @@
             <artifactId>cache-api</artifactId>
             <version>1.0.0</version>
         </dependency>
+        <dependency>
```

Comment:
Can you make sure to run owasp on this module to make sure these new dependencies don't have any vulnerabilities.